### ISSUE TITLE
Improve contrast of code listings

### ISSUE THREAD

**Issue by LiuSuY** [yeoman/yeoman.io#747]
# 官方地址 http://yeoman.io/learning/
# First thing is to install yo using npm:
   下面的命令行
   离电脑稍微远一点点时 只能看见一个黑块 不仔细看都无法发现里面还有命令

**Comment by mischah** [comment]
Sorry, please open issues using english. Otherwise the maintainers of Yeoman won’t be able to help you 😞

**Comment by mischah** [comment]
Are we open? 
I personally don’t fell the need. And my contrast ratio checker says it’s a AAA contrast:
![image](https://user-images.githubusercontent.com/441011/32941157-24921ad6-cb85-11e7-84da-c1c29bd942ae.png)

What’s your opinion?

**Comment by LiuSuY** [comment]
Yeah, that's what I want

**Comment by mischah** [comment]
We might want switch to [Prism](http://prismjs.com/) since it’s more lightweight than highlight.js

**Comment by mischah** [comment]
Guess we can call this an »actionable«.

**Comment by mischah** [comment]
Oh. interesting. It looks like that on my machine:
![image](https://user-images.githubusercontent.com/441011/32943799-9f091622-cb8d-11e7-8756-c2d0f653e8ce.png)

**Comment by LiuSuY** [comment]
Yes, it's fun, and you can see it clearly even on the back of your seat
是的,很有趣，即使靠在座椅的做背上也能看得很清晰

**Comment by SBoudrias** [comment]
@LiuSuY 上网的时候你用 Chrome 了吗。

**Comment by LiuSuY** [comment]
不用chrome 或者火狐难道用ie?
Doesn't chrome or Firefox use IE?

是的，我正在使用
Yes, I'm using it

**Comment by JoshuaKGoldberg** [comment]
The code snippets have very good contrast for me. I don't see any that don't have at least AA levels. Closing out this old issue as aged away, and separately filing #848.
